### PR TITLE
Add missing '\param' documentation

### PR DESCRIPTION
**Pull request by 1bsyl**

### CODE REVIEW

File: include/SDL3/SDL_hidapi.h [hunk 1]
```diff
@@ -235,6 +235,7 @@ extern DECLSPEC SDL_hid_device * SDLCALL SDL_hid_open(unsigned short vendor_id,
  * platform-specific path name can be used (eg: /dev/hidraw0 on Linux).
  *
  * \param path The path name of the device to open
+ * \param bExclusive exclusive opening (windows only?FIXME)
```
Comment (sezero) [inline include/SDL3/SDL_hidapi.h:238]: Yes, `bExclusive` is for windows only. Documentation may be changed
to something like this?
`* \param bExclusive Open device in exclusive mode (Windows only)`

File: include/SDL3/SDL_joystick.h [hunk 2]
```diff
@@ -309,6 +309,10 @@ extern DECLSPEC SDL_Joystick *SDLCALL SDL_GetJoystickFromPlayerIndex(int player_
 /**
  * Attach a new virtual joystick.
  *
+ * \param type Type of joystick
+ * \param naxes number of axes
+ * \param nbuttons number of buttons
+ * \param nhats number of hats
```
Comment (sezero) [inline include/SDL3/SDL_joystick.h:315]: Incosistent first letter capitalization, e.g. `Type` vs `number` .   Same in several other places too.

File: include/SDL3/SDL_metal.h [hunk 3]
```diff
@@ -73,6 +74,8 @@ extern DECLSPEC SDL_MetalView SDLCALL SDL_Metal_CreateView(SDL_Window * window);
  * This should be called before SDL_DestroyWindow, if SDL_Metal_CreateView was
  * called after SDL_CreateWindow.
  *
+ * \param view the view
```
Comment (sezero) [inline include/SDL3/SDL_metal.h:77]: Maybe change to `The SDL_MetalView object` ??

File: include/SDL3/SDL_surface.h [hunk 4]
```diff
@@ -867,6 +879,8 @@ extern DECLSPEC int SDLCALL SDL_BlitSurfaceUncheckedScaled
 /**
  * Set the YUV conversion mode
  *
+ * \param YUV conversion mode
```
Comment (sezero) [inline include/SDL3/SDL_surface.h:882]: Wrong parameter name here. Maybe change to this:
` * \param mode YUV conversion mode`

Comment (1bsyl) [reply include/SDL3/SDL_surface.h:882]: was fixed

**Comment by 1bsyl** [comment]
I adding the fixes here ...

File: include/SDL3/SDL_metal.h (hunk 3 above)
Comment (1bsyl) [reply include/SDL3/SDL_metal.h:77]: done in https://github.com/libsdl-org/SDL/pull/7300

File: include/SDL3/SDL_joystick.h (hunk 2 above)
Comment (1bsyl) [reply include/SDL3/SDL_joystick.h:315]: done in https://github.com/libsdl-org/SDL/pull/7300
only this one.
indeed this is not consistent accross various SDL comment

**Pull request merged**